ompi/mpi/java/java/Comm.java

```diff
@@ -1185,7 +1185,7 @@ private native void setErrhandler(long comm, long errhandler)
 
 	/**
 	 * Returns the error handler currently associated with the communicator.
-	 * <p>Java binding of the MPI operation {@code MPI_ERRHANDLER_GET}.
+	 * <p>Java binding of the MPI operation {@code MPI_COMM_ERRHANDLER_GET}.
```

Comment:
This should be ` MPI_COMM_GET_ERRHANDLER`.

Reply:
I fixed it.